<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>784</width>
    <height>496</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="Send_Button">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>450</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>SEND</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="Chat_Block">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>450</y>
     <width>421</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Search_Button">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>360</y>
     <width>71</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Login_Button">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>50</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </widget>
  <widget class="QComboBox" name="Chat_Chart">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>181</width>
     <height>61</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>alone</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>study</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>movie</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>book</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>hobby</string>
    </property>
   </item>
  </widget>
  <widget class="QTextEdit" name="Chat_Area">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>70</y>
     <width>531</width>
     <height>371</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="Chat_Object">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>400</y>
     <width>181</width>
     <height>26</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>채팅방 이름</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="Join_Button">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>440</y>
     <width>61</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>join</string>
   </property>
  </widget>
  <widget class="QPushButton" name="NAME">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>NICKNAME</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="NickName_Block">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="Chat_Name">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>20</y>
     <width>531</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="NickName">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="Search_Block">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Search_Text">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>440</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
